ingest/user_de.py

```diff
@@ -170,15 +177,31 @@ def generate_individual_files(col, genes, rest, groups, clean_val, qual):
 
 # final result: individual files for each comparison
 
+def generate_manifest(clean_val, qual):
+    """
+    create manifest file of each comparison in the initial data
+    if the comparison is with rest, rest is omitted and just the type is written
+    """
+    file_names = []
+    for i in clean_val:
+        cleaned_list = [ x for x in i if x != "rest" and x not in qual]
+        file_name = '_'.join(cleaned_list)
+        if file_name not in file_names:
+            file_names.append(file_name)
+
+    with open('ingest/manifest.tsv', 'w', newline='') as f:
+        tsv_output = csv.writer(f, delimiter='\t')
+        tsv_output.writerow(file_names)
+
 if __name__ == '__main__':
     data = pd.read_csv(file_path)
     long_format = convert_wide_to_long(data)
     wide_format = convert_long_to_wide(long_format)
-
     data_by_col = get_data_by_col(data)
     col, genes, rest = data_by_col
-
     groups_and_props = get_groups_and_properties(col)
     groups, clean_val, qual = groups_and_props
+    generate_manifest(clean_val, qual)
+    generate_individual_files(col, genes, rest, groups, clean_val, qual)
```

Comment:
Was using the correct format for for the individual filenames supposed to be a part of this PR, or was that for subsequent work to push the file back to the bucket?  Whenever we decide that work needs to be done, the filenames need to follow [this structure](https://github.com/broadinstitute/scp-ingest-pipeline/blob/development/ingest/de.py#L393).
```
out_file = f'{cluster_name}--{clean_annotation}--{clean_group}--{annot_scope}--{method}.tsv'
```